core/src/test/java/org/elasticsearch/search/aggregations/InternalAggregationTestCase.java

```diff
@@ -122,4 +136,49 @@ protected final T createUnmappedInstance(String name) {
     protected NamedWriteableRegistry getNamedWriteableRegistry() {
         return namedWriteableRegistry;
     }
+
+    public void testFromXContent() throws IOException {
```

Comment:
cool! this means that we don't have to write unit tests for each single agg anymore, as this one is generic enough?